src/App.css

```diff
@@ -1,31 +1,235 @@
+  color: #000;
+}
+
+.categories-link {
+  color: #121212;
+}
+
+.bookInfo {
+  padding-left: 10%;
+  width: 80%;
+  border: 1px solid #e8e8e8;
+  margin-bottom: 20px;
   justify-content: space-between;
+  align-items: center;
+}
+
+.Remove {
+  width: 10rem;
+  height: 5vh;
+  background-color: #0290ff;
+  color: #fff;
+  border-radius: 3px;
+  border: none;
+}
+
+.author {
+  color: #4386bf;
+  text-align: center;
+}
+
+.category {
+  width: 7rem;
+  height: 1.125rem;
+  margin: 0 4rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
 }
 
 .book-form {
-  width: 10%;
-  display: flex;
-  flex-direction: column;
+  padding-left: 10%;
+  width: 80%;
 }
 
 input {
+  width: 20rem;
+  height: 3rem;
+  margin-right: 20px;
+}
+
+.Add {
+  width: 10rem;
+  height: 3.3rem;
+  background-color: #0290ff;
+  color: #fff;
+  font-weight: 700;
+  border-radius: 3px;
+  border: none;
+}
+
+.li-buttons {
+  background: none !important;
+  border: none;
+  width: 5.563rem;
+  height: 1.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border-right: 1px solid;
+}
+
+.li-buttons:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.book-author {
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 30px;
+}
+
+.wrapper {
+  width: 100px;
+  height: 100px;
+  clip: rect(0, 100px, 100px, 50px);
+  position: relative;
+  left: -96%;
+}
+
+.circle {
+  width: 80px;
+  height: 80px;
+  border: 10px solid #4386bf;
+  border-radius: 50px;
+  position: absolute;
+  clip: rect(0, 50px, 100px, 0);
+}
+
+div[data-anim~=base] {
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+  animation-timing-function: linear;
+}
+
+.wrapper[data-anim~=wrapper] {
+  animation-duration: 0.01s;
+  animation-delay: 1s;
+  animation-name: close-wrapper;
+  position: relative;
+  left: 0;
+}
+
+.pres {
+  width: 2.938rem;
+  height: 2.438rem;
+  margin: 0.063rem 7.063rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  color: #121212;
+}
+
+.presentage {
+  flex-direction: column;
+  width: 50%;
+}
+
+.left {
+  margin-top: 20px;
+  width: 30%;
+  justify-content: center;
+  align-items: center;
+}
+
+.completed {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  color: #121212;
   margin-bottom: 10px;
 }
+
+.border {
+  width: 30%;
+  height: 13vh;
+  border-left: 1px solid gray;
+  flex-direction: column;
+  margin-right: 30px;
+  align-items: center;
+  justify-content: center;
+}
+
+.lesson {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  margin-bottom: 40px;
+}
+
+#bookInput {
+  width: 40%;
+}
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![Screen Shot 2021-08-26 at 00 15 40](https://user-images.githubusercontent.com/45217364/151683647-9a7effd3-0c43-41b6-8d45-df8fd8858655.png) |

  - In the mockup design, the app is more like a card in the center of the screen.
  - You still have many margins gaps that aren't in the mockup design. Kindly address them.
  - Remember to make sure that left and right margins are the same. in other words, The card must be in the center of the horizontal orientation.